pytube/streams.py

```diff
@@ -61,6 +63,15 @@ def __init__(
 
         # filesize in bytes
         self._filesize: Optional[int] = int(stream.get('contentLength', 0))
+        
+        # filesize in kilobytes
+        self._filesizekb: Optional[float] = float(ceil(stream.get('contentLength', 0) / 1024 * 1000) / 1000)
```

Comment:
```suggestion
        self._filesize_kb: Optional[float] = float(ceil(stream.get('contentLength', 0) / 1024 * 1000) / 1000)
```
could we add another _ here to delimit

Reply:
and use _ more generally to delimit words in the function names and variables (via https://peps.python.org/pep-0008/#function-and-variable-names)